buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -25,9 +25,9 @@ object Versions {
     internal const val ASSERTJ = "3.20.2"
     internal const val COIL = "1.3.0"
     internal const val COMPOSE_ACCOMPANIST = "0.14.0"
```

Comment:
This will probably happen today/tomorrow too, so we can be adamant of getting it up to date tomorrow I guess